python_scripts/database_builder_v2.py

```diff
@@ -605,21 +588,7 @@ def rebuild_database(db_name, rebuild_db, delete_underlying_data=True):
     glue_client = boto3.client("glue")
 
     # Delete and re-create database
-    rebuild_database(db_name, True)
-
-    # Drop database
-    # glue_client.delete_database(Name=db_name)
-
-    # Create the database
-    # glue_client.create_database(DatabaseInput={
-    #                                 'Name': db_name,
-    #                                 'Description': f'Database for storing tables extracted from the matrix booking system API: {args.env} environment',
-    #                                 'LocationUri': f'alpha-dag-matrix/db/{args.env}/'
-    #                                     }
-
-    #                            )
-
-    # Add tables
+    rebuild_database(database_name="matrix_db_dev", rename_db=db_name)
```

Comment:
Why is the database name hardcoded?

Reply:
I could not figure out how to pass database_name as a separate variable. I was able to get it working enough to rename the dev db, I didn't mean to commit it in the state it was in.